onnxruntime/contrib_ops/cpu/tokenizer.cc

```diff
@@ -246,40 +77,36 @@ Tokenizer::Tokenizer(const OpKernelInfo& info) : OpKernel(info) {
   status = info.GetAttrs("separators", separators);
   if (!status.IsOK()) {
     status = info.GetAttr("tokenexp", &tokenexp);
-    ORT_ENFORCE(status.IsOK(), "Either one of the separators OR tokenexp attributes required but none is set");
+    ORT_ENFORCE(status.IsOK(), "Either one of the sepexp OR tokenexp attributes required but none is set");
```

Comment:
Change back to separators

Reply:
sepexp should be changed back to 'separators'?

Reply:
As we discussed we will not change then.

---
In reply to: [276857608](https://github.com/Microsoft/onnxruntime/pull/857#discussion_r276857608) [](ancestors = 276857608)